pitop/processing/core/vision_functions.py

```diff
@@ -99,20 +99,21 @@ def get_object_target_lock_control_angle(center, frame):
     :param frame:
             OpenCV frame that has the same scale used for center parameter - this function uses the dimensions of
             the frame.
-    :return:
+    :return float angle: Angle in degrees to 1 decimal place
     """
-    from numpy import arctan, pi
+    from numpy import arctan
+    from math import degrees
     if center is None:
         return None
     # physically, this represents an approximation between chassis rotation center and camera
     # the PID loop will deal with basically anything > 1 here, but Kp, Ki and Kd would need to change
     # with (0, 0) in the middle of the frame, it is currently set to be half the frame height below the frame
     chassis_center_y = -int(frame.shape[1])
 
-    # Anticlockwise is positive angle
+    # Clockwise is positive angle
```

Comment:
Won't inverting the behaviour here affect the line follower?

Reply:
Just the comment was wrong - hence why comments are bad :D